examples/spire-sidecar/config/apps.bash.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: path-to-inputfile
+data:
+  inputfile.txt: |
+    db-config/config.ini
+    db-config/config.json
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: apps
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: apps
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        identity_template: "true"
+        app: apps
+    spec:
+      initContainers:
+        - name: sidecar
+          securityContext:
+            # privilaged is needed to create socket and bundle files
+            privileged: true
+          image: tsidentity/tornjak-example-sidecar:v0.1
+          imagePullPolicy: Always
+          # use command to override the command of the image/dockerfile
+          command: ["/usr/local/bin/run-sidecar-bash.sh"]
+          args:
+            - "/usr/local/bin/inputfile.txt"
+          env:
+          - name: SOCKETFILE
+            value: "/run/spire/sockets/agent.sock"
+          - name: ROLE
+            value: "dbrole"
+          - name: VAULT_ADDR
+            value: "http://tsi-vault-tsi-vault.space-x04-9d995c4a8c7c5f281ce13d5467ff6a94-0000.eu-de.containers.appdomain.cloud"
```

Comment:
Once we are done with testing, we should remove all the references to our test cluster. We have to document the README that Vault address need to be provided before execution.